cmd/kola/options.go

```diff
@@ -142,6 +142,7 @@ func init() {
 	sv(&kola.QEMUOptions.Board, "board", defaultTargetBoard, "target board")
 	sv(&kola.QEMUOptions.DiskImage, "qemu-image", "", "path to CoreOS disk image")
 	sv(&kola.QEMUOptions.BIOSImage, "qemu-bios", "", "BIOS to use for QEMU vm")
+	bv(&kola.QEMUOptions.NoSwtpm, "no-swtpm", false, "Don't create a temporary software TPM")
```

Comment:
Hmm, might be better to not make this a negative flag but rather something like `qemu-swtpm` defaulting to `true`; this would also have the added benefit of directly showing it's tied to the `qemu` related platforms.

Reply:
will do.